util/sdkutil/events.go

```diff
@@ -0,0 +1,14 @@
+package sdkutil
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/gogo/protobuf/proto"
+)
+
+// Emit proto event and log on error
+func Emit(ctx *sdk.Context, event proto.Message) {
```

Comment:
Why we need `*sdk.Context` , you can move this to `sdk.Context`  and can you rename `Emit` to `EmitEvents`

Reply:
> Why we need *sdk.Context ,

Not needed. This way we avoid coping. sdk.Context is rather a "medium" level structure.

Reply:
> you can move this to sdk.Context 

Move what to the `sdk.Context`?

> and can you rename Emit to EmitEvents

I prefer short names. I don't think there is anything else we emit than events.